NEWS

```diff
@@ -12,6 +12,10 @@
   which have empty repository scopes (gh#SUSE/machinery#1268)
 * Inspection of unmanaged-files is now also using the faster machinery-helper
   when the files are extracted
+* Introduce format version 5 which adds the "environment" scope
```

Comment:
@greygoo I would also suggest to use backticks instead of quotes. It seems that we're using more backticks than quotes. Am I right?